autoscale/src/main/java/com/sequenceiq/periscope/service/EntitlementValidationService.java

```diff
@@ -48,4 +56,13 @@ public boolean stopStartAutoscalingEntitlementEnabled(String accountId, String c
         }
         return entitled;
     }
+
+    public List<String> impalaScheduleBasedScalingEntitlementEnabled(String accountId, CloudPlatform cloudPlatform) {
```

Comment:
rename it to impalaScheduleBasedScalingMissingEntitlements

Reply:
Done